### ISSUE TITLE
Latest website version?

### ISSUE THREAD

**Issue by elimisteve** [radii/noisetor-www#1]
Hi there! Now that noisetor.net points to a server I can shell into, I'm setting up the homepage once again. (Apparently noisetor.net was being hosted on one of our exit node servers that were closed down when Noisetor was handed over to me...)

Do you happen to know if this repo contains the very latest version of the noisetor.net homepage? Thanks!

**Comment by radii** [comment]
IIRC the noisetor website was always hosted at iocoop on a VM on my hardware there. The hardware died and the noisetor vm was one that I didn't revive onto my new (extra jankified) setup.
this repo probably contains the most recent web content but I'll have to dig up the archived FS image to confirm that.

**Comment by radii** [comment]
@elimisteve thanks for taking over the noisetor project, much appreciated. What's the vision for the project? are you (the new project owners) going to resume operating relays / exit-nodes? Do you want to re-invigorate donations to pay for additional costs? Maybe this issue isn't the right place to discuss, idk...

**Comment by radii** [comment]
yes I found the filesystem image and this repo has the most recent web content.

**Comment by elimisteve** [comment]
> yes I found the filesystem image and this repo has the most recent web content.

Thanks!

**Comment by elimisteve** [comment]
> @elimisteve thanks for taking over the noisetor project, much appreciated. What's the vision for the project? are you (the new project owners) going to resume operating relays / exit-nodes? Do you want to re-invigorate donations to pay for additional costs? Maybe this issue isn't the right place to discuss, idk...

You bet! Hell yes re: exit node(s), and probably yes re: donations. There's already probably significant money set aside for Noisetor within Noisebridge, although that's not completely clear. **Do you happen to have the Python script that Patrick O has alluded to that grabs NB donation data from PayPal's API, particularly the donations tagged with "noisetor"?** We need that to know whether Noisetor has money or not.

Vision/next steps: I've thought or years that it'd be great to create a tool that makes it dramatically simpler for people to create a host to run their Tor server (exit or otherwise). I imagine a really simple-to-use web app that helps people choose a hosting provider (or tell them to donate to Noisetor or similar if they can't afford a sufficiently beefy server) based upon which highly-rated, pro-Tor providers are most under-utilized, and based on which ASes are most under-utilized. I'd like to ask Dingledine about this. **What do you think is the best way to contact Dingledine for this purpose?**

Thanks!

P.S. I'm keybase.io/elimisteve and my PGP key etc are at https://tryingtobeawesome.com/contact/ if you'd prefer that.

**Comment by elimisteve** [comment]
@radii Think the Noisetor payment-parsing Python script may be around some place?
